features/local-network-access.yml

```
@@ -0,0 +1,5 @@
+name: Local network access
+description: Restricts the ability to send requests to servers on a user's local network and requiring the user grant the site permission before such requests can be made.
```

Interesting! [This section in the spec](https://wicg.github.io/local-network-access/#rollout-difficulties) makes me wonder if this _is_ a developer facing feature at all:

> Local Network Access essentially deprecates direct access to the local network in favor of more secure user-agent-mediated alternatives. Web deprecations are hard. Chromium previously encountered many stumbling blocks on the way to shipping parts of [[PRIVATE-NETWORK-ACCESS]](https://wicg.github.io/local-network-access/#biblio-private-network-access) (the predecessor to this specification).

I guess my question here, is what's the developer-facing entry point? Is it [the permission](https://wicg.github.io/local-network-access/#integration-with-permissions) perhaps? Unless I'm missing something, it seems like maybe the only "feature" here is something like…


```suggestion
description: The `"local-network-access"` user permission (and certain actions that imply this permission, such as a `fetch()` request with `{ targetAddressSpace: "local" }`) allows a site to send request to servers on a user's local network.
```